Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_PlayerPhysics.cs

```diff
@@ -0,0 +1,115 @@
+namespace VRTK
```

Comment:
As you're thinking of getting rid of this and using/updating the player presence script. I won't bother reviewing it yet,